libraries/CurieBLE/examples/LEDCentral/LEDCentral.ino

```diff
@@ -0,0 +1,157 @@
+/*
+  Copyright (c) 2016 Intel Corporation. All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-
+  1301 USA
+*/
+
+// This example can work with CallbackLED to show the profile read/write operation in central
```

Comment:
I have tested this example with CallbackLED. I think every sketches has a goal. I have designed those examples, Scan, establish connection, read/write profile and process notification. For the peripheral, I think we have some APP to test it. Do you think we need align the peripheral and central?